src/dotnet/APIView/APIViewWeb/Managers/CodeFileManager.cs

```diff
@@ -196,11 +198,25 @@ public async Task<APICodeFileModel> CreateReviewCodeFileModel(string apiRevision
         /// <param name="codeFileA"></param>
         /// <param name="codeFileB"></param>
         /// <returns></returns>
-        public bool AreAPICodeFilesTheSame(RenderedCodeFile codeFileA, RenderedCodeFile codeFileB)
+        public async Task<bool> AreAPICodeFilesTheSame(RenderedCodeFile codeFileA, RenderedCodeFile codeFileB)
```

Comment:
because this is an azync function, the method name should have `async` at the end `AreAPICodeFilesTheSameAsync`